docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -206,35 +206,53 @@ V&V Checks:
 
 
 .. list-table:: Intrapartum Decision Tree
-  :widths: 3 15 15
+  :widths:  3 5 10 10 15
   :header-rows: 1
 
   * - ID
     - Decision Information 
+    - Data Value 
+    - Source
     - Notes
   * - 0
-    - XX% of simulants have ectopic pregnancies, abortion or miscarriage
-    - These simulants will NOT continue in the model
+    - Incidence of ectopic pregnancies, abortion or miscarriage
+    - get_draws(gbd_round_id=7, location_id=location_id, gbd_id_type='cause_id', gbd_id=[995,374], source='como', measure_id=6, metric_id=3, age_group_id=24, sex_id=2, year_id=2021, decomp_step='iterative')
+    - incidence_c374 + incidence_c995
+    - These simulants will NOT continue in the model. Use the `total population incidence <Total Population Incidence Rate>`_ rate directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence.
   * - 1
-    - XX% of simulants to attend each delivery facility type, based on their propensity 
-    - Several details are still outstanding including: types of delivery facilities modeled, will facility propensity vary with age, subnational location or upstream factors, will ANC care propensity be correlated with delivery facility propensity
+    - % of simulants to attend each delivery facility type, based on their propensity
+    - At home (68.3%), in public/governmental health facility (26.6%), in private and/or NGO health facility (3.3%), and other (4.8%) 
+    - DHS for each location; placeholder values are from `this Ethiopia paper <https://link.springer.com/article/10.1186/s12884-020-03002-x#Tab2>`_.
+    - Denominator in DHS is all births (live and stillbirths) to interviewed women in the 2 years preceding the survey. The above values are placeholders until we do a more in-depth analysis. We would like this to be location specific, please code accordingly. 
   * - 2
     - Need to figure out how we will determine which simulants need a c-section
+    -
     - 
+    -    
   * - 3
-    - XX% of each facility type have cesarian section capabilities
-    -  
+    - % of each facility type have cesarian section capabilities
+    - Public/governmental health facility (62%), private and/or NGO health facility (73%), at home  (0%), other (50%) 
```

Comment:
How should I think about "other"?  Is this facility type common?

Reply:
Yes, I should have made a comment about this also! 'Other' signifies any health facility that doesn't fall into one of the four standard categories (public hospital, private for-profit, NGO, or at home -- these four categories were updated in [my second PR](https://github.com/ihmeuw/vivarium_research/pull/1533) after I had a better sense of data availability) and includes what is coded as 'country-specific government health facility' in DHS. For Ethiopia, for example, I think this includes [health posts or Health Extension Workers](https://l10k.jsi.com/Health/hep.htm#:~:text=Health%20posts%20are%20located%20at,health%20extension%20workers%20(HEWs).). I still need to do more investigating as to what this would entail for Pakistan and Nigeria.  I can clarify the wording in this to make this more clear: e.g., change 'Public/governmental health facility' to 'Public/governmental hospital' and 'other' to 'country-specific government health facility'.

Reply:
> I can clarify the wording in this to make this more clear: e.g., change 'Public/governmental health facility' to 'Public/governmental hospital' and 'other' to 'country-specific government health facility'.

Do you think this wording would be more accurate? It doesn't look like you made this change in your other PR.